.github/workflows/_gen_workflows.py

```diff
@@ -37,21 +37,22 @@ class BaseTests:
         ],
     }
 
-    run_tests = {
-        "name": "Test",
+    run_checks = {
+        "name": "Test, lint and docs",
         "env": {
             "GITHUB_ACTIONS_MATRIX_NAME": "${{ matrix.name }}",
             "CI_CREDENTIALS": "${{ secrets.CI_CREDENTIALS }}",
             "CI_ENDPOINT": "${{ secrets.CI_ENDPOINT }}",
         },
-        "run": "python -m invoke test",
+        "run": "inv check-all",
     }
 
     def __init__(self):
         project_dir = CURDIR.parent.parent / self.project_name
         assert project_dir.exists(), f"{project_dir} does not exist"
 
         self.target_file = CURDIR / f"{self.target}"
+        self.devutils_requirements = CURDIR / "devutils" / "requirements.txt"
```

Comment:
Won't this point out to: _.github/workflows/devutils/requirements.txt_ by mistake?

Maybe we want here:
```python
self.devutils_requirements = CURDIR.parent.parent / "devutils" / "requirements.txt"
```
as seen with `project_dir`.